term/functions.php

```diff
@@ -34,6 +34,18 @@ function term_block_styles() {
 				}',
 			)
 		);
+
+		register_block_style(
+            'core/image',
+            array(
+                'name' => 'bordered',
+                'label' => __('Bordered', 'term'),
+                'inline_style' => '.wp-block-image.is-style-bordered img{
```

Comment:
I don't think we should do this because we can already use the border controls on the image block. We only did it for the social icons because those controls are not there yet

Reply:
The reason being: a user can only opt in or out of the border radius, they can't tweak it to other values

Reply:
Yeah, you're right, don't think about it, sorry.